docs/static/style-guide/formats.md

```diff
@@ -0,0 +1,197 @@
+> This library provides various functions for formatting the data in the preferred style.
+
+## Formatting Functions
+
+#### `sszvis.formatText(x)`
+
+Any given data is returned as a string and will be formatted as Text.
+
+```table
+rows:
+  - Function: formatText
+    Example: sszvis.formatText(16000)
+    Result: "'16000'"
+  - Function:
+    Example: sszvis.formatText("Jahr")
+    Result: "'Jahr'"
+```
+
+#### `sszvis.formatNumber(x)`
```

Comment:
Please also add some examples that show how numbers will be shortened if they have insignificant decimals. See `test/format.test.js` for more examples.